server/public/views.py

```diff
@@ -19,3 +20,6 @@ def list(self, request, *args, **kwargs):
             data={"id": request.GET.get("id")},
             status=HTTP_200_OK
         )
+
+    def create(self, request, *args, **kwargs):
+        return Response(status=HTTP_200_OK)
```

Comment:
yes typically, but since we don't create anything..

Reply:
Haha true. I think just for "best practices" sake, write a comment saying "Create your obj" above the `return` and then still change the status code to 201.

It might also be worth it, for best practices sake, to simply import all status:

```
from rest_framework import status
```

And then everything else would be `status.HTTP_200_OK` and `status.HTTP_201_CREATED`. Thanks!

Reply:
yes